src/App.elm

```diff
@@ -91,17 +97,14 @@ update msg model =
                             Nothing ->
                                 LoggedOut
 
-                    ( fetchMyUser, fetchConversationList ) =
+                    fetchMyUser =
                         case info of
                             Just loginInfo ->
-                                ( fetchMe loginInfo
+                                fetchMe loginInfo
                                     |> Task.perform FetchMeFailed FetchMeLoaded
-                                , fetchConversations loginInfo
-                                    |> Task.perform FetchConversations FetchConversations
-                                )
 
                             Nothing ->
-                                ( Cmd.none, Cmd.none )
+                                Cmd.none
```

Comment:
In these cases we can get rid of the `case` using `Maybe.withDefault`

```elm
fetchMyUser =
	info
		|> Maybe.map (fetchMe >> Task.perform FetchMeFailed FetchMeLoaded)
		|> Maybe.withDefault Cmd.none
```

I think that the above will work, but it might need some reworking